### DISCUSSION TITLE
Accept all confirmations mode

### DISCUSSION THREAD

**Discussion by TheAB1** [dyc3/steamguard-cli#358] [Ideas]
Although it may result in a security issue, a special mode to automatically accept all incoming confirmations would be useful, especially when using trading bots. Have you thought about adding this feature?

**Comment by dyc3** [comment]
If you are using a trading bot, the bot should be the one responsible for accepting confirmations. That's what I did with my bot when I was trading. 

There is already `steamguard trade --accept-all`. I don't plan to add a mode where you leave it running an it accepts all the confirmations. You can do that already with a bash while loop or whatever the equivalent is in powershell.
